extensions/wikia/AdEngine/js/video/videoFrequencyStore.js

```diff
@@ -0,0 +1,103 @@
+/*global define*/
+define('ext.wikia.adEngine.video.videoFrequencyStore', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adLogicPageViewCounter',
+	'ext.wikia.adEngine.utils.time',
+	'wikia.cache',
+	'wikia.log'
+], function (adContext, pageViewCounter, timeUtil, cache, log) {
+	'use strict';
+
+	var cacheKey = 'adEngine_outstreamVideoFrequency',
+		cacheTtl = cache.CACHE_LONG, // 1 month
```

Comment:
Do we need so long cache for video frequency capping? I believe it's ok to show video ad again to a person on the next day.